graphix/transpiler.py

```diff
@@ -602,6 +675,8 @@ def _move_byproduct_to_right(self):
                 continue
             if self._instr[target + 1][0] == "CNOT":
```

Comment:
we'll need to add _commute_with_ccx, to be complete. but I am thinking of discontinuing standardize_and_tranpile, so we might skip that for now

Reply:
@masa10-f shall we discontinue `standardise_and_transpile`? I think there isn't much of benefit (#25).